internal/http/services/owncloud/ocdav/copy.go

```diff
@@ -153,7 +163,7 @@ func (s *svc) handleCopy(w http.ResponseWriter, r *http.Request, ns string) {
 		// TODO what if intermediate is a file?
 	}
 
-	err = descend(ctx, client, srcStatRes.Info, dst)
+	err = descend(ctx, client, srcStatRes.Info, dst, depth == "infinity")
```

Comment:
Shouldn't we descend also in depth > 1?

Reply:
the spec doesn't allow any other values than "0" and "infinity": https://tools.ietf.org/html/rfc4918#section-9.8.3

Reply:
@PVince81 Depth 1 is used by the sync clients